plugin/trino-hive/src/main/java/io/trino/plugin/hive/NoneHiveMaterializedViewMetadata.java

```diff
@@ -50,4 +51,16 @@ public MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession s
     {
         throw new TrinoException(NOT_FOUND, "This connector does not support materialized views");
     }
+
+    @Override
+    public boolean delegateMaterializedViewRefreshToConnector(ConnectorSession session, SchemaTableName viewName)
+    {
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support dropping materialized views");
```

Comment:
nit: should be `This connector does not support materialized views` in both methods

Reply:
created https://github.com/trinodb/trino/pull/8228